recipe/patch_yaml/tpot.yaml

```
@@ -0,0 +1,31 @@
+# uses now-removed `imp` module, see:
+# - https://github.com/conda-forge/tpot-feedstock/issues/39
+# - https://github.com/EpistasisLab/tpot/issues/1327
+# noarch was added on
+# - https://github.com/conda-forge/tpot-feedstock/pull/21
+if:
+  name: tpot
+  version_ge: 0.11.2
```

It looks like `0.11.0` & `0.11.1` were `noarch` as well. Should we include them too?

```suggestion
  version_ge: 0.11.0
```

---

will check if that blends against py3.11...